stix_shifter_modules/msatp/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -81,9 +396,14 @@
         "object": "nc_process_file"
       },
       {
-        "key": "process.image_ref",
+        "key": "process.binary_ref",
```

Comment:
it should be process.image_ref. stix 2.1 doesn't have a binary_ref property for process object.
(basically all these `binary_ref` mapping need to revert back to `binary_ref`

Reply:
fixed